sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -668,7 +668,7 @@
       "delete": {
         "summary": "Deletes all the pending commands for this device from the IoT hub.",
         "description": "Deletes all the pending commands for this device from the IoT hub.",
-        "operationId": "RegistryManager_PurgeCommandQueue",
+        "operationId": "Messages_PurgeCommandQueue",
```

Comment:
PurgeCouldToDeviceMessages; it's not really a command

Reply:
+1, this is a big point of confusion. "PurgeCloudToDeviceMessageQueue" to be more explicit that it is a queue